src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -140,6 +141,8 @@ object ScalafixPlugin extends AutoPlugin {
         loadedRules = () => scalafixArgs().availableRules().asScala,
         terminalWidth = Some(JLineAccess.terminalWidth)
       ).parser.parsed
+      val errorLogger =
+        new PrintStream(LoggingOutputStream(streams.value.log, Level.Error))
```

Comment:
I thought about deferring the logging until the point we know the status code, to adjust the level accordingly, but looking at the usage of that `printStream`, I think it's reasonnable to always log as error, except *maybe* for [`--stdout`](https://github.com/scalacenter/scalafix/blob/1564a4b7eec87e16f2c6bce233848c6b542eb966/scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala#L262), but that's a lot of custom code for something that I don't see being used through sbt